x-pack/plugins/task_manager/server/task_manager.ts

```diff
@@ -240,7 +240,7 @@ export class TaskManager {
    * @param taskDefinitions - The Kibana task definitions dictionary
    */
   public registerTaskDefinitions(taskDefinitions: TaskDictionary<TaskDefinition>) {
-    this.assertUninitialized('register task definitions');
+    this.assertUninitialized('register task definitions', Object.keys(taskDefinitions).join(', '));
```

Comment:
When the migration caused lifecycle problems it was very hard to figure out which task was causing the issues.
Changing this message made it easier to pin point the source of the issue and it seemed worth keeping this change.